itou/www/employee_record_views/forms.py

```diff
@@ -119,7 +119,7 @@ class NewEmployeeRecordStep1Form(forms.ModelForm):
     birth_place = forms.ModelChoiceField(
         queryset=Commune.objects,
         label="Commune de naissance",
-        help_text="La commune de naissance ne doit être saisie que lorsque le salarié est né en France",
+        help_text="La commune de naissance est obligatoire lorsque le salarié est né en France",
```

Comment:
Les deux infos me semblent importantes.

```suggestion
        help_text="La commune de naissance est obligatoire lorsque le salarié est né en France et ne doit pas être saisie dans le cas contraire",
```

Reply:
Pour quelle raison on devrait garder "ne doit pas être saisie dans le cas contraire" ?

Reply:
Je n’ai pas discuté avec Xavier, mais j’imagine que c’est pour que les utilisateurs nés en dehors de France sachent que le système attend qu’ils ne remplissent pas ce champ du formulaire.